apps/class-solid/src/lib/experiment_config.ts

```diff
@@ -42,6 +56,28 @@ const jsonSchemaOfExperimentConfigBase = {
       },
       default: [],
     },
+    observations: {
+      type: "array",
+      items: {
+        type: "object",
+        properties: {
+          height: { type: "array", items: { type: "number" } },
+          pressure: { type: "array", items: { type: "number" } },
+          temperature: { type: "array", items: { type: "number" } },
+          relativeHumidity: { type: "array", items: { type: "number" } },
+          windSpeed: { type: "array", items: { type: "number" } },
+          windDirection: { type: "array", items: { type: "number" } },
+        },
+        required: [
+          "height",
```

Comment:
And if you decide to make it required also add name here.

```suggestion
          "name",
          "height",
```